src/api/index.tsx

```diff
@@ -202,7 +207,13 @@ export const callApi = async <
       );
     }
 
-    return await response.json();
+    const contentType = (
+      response.headers.get('content-type') || ''
+    ).toLowerCase();
+
+    return await (contentType.startsWith('application/json')
```

Comment:
Given that we're calling a JSON API, I think it'd be handy to mention that this is to handle `204` responses, as described in https://github.com/whatwg/fetch/issues/113

Reply:
Well, it's really just any text response, not necessary a 204. Also, I agree with the closure of that issue: it's just not valid json when it's text so an error is expected.

Reply:
Sure, I could add a comment about why the API isn't returning json sometimes

Reply:
thanks